### PR TITLE
server.py: handle possible case where only 1 completion causes error

### PR DESCRIPTION
**Pull request by mallochine**

### CODE REVIEW

**Comment by Shougo** [comment]
Merged.

File: rplugin/python3/deoplete/sources/deoplete_jedi/server.py [hunk 1]
```diff
@@ -309,7 +309,10 @@ def script_completion(self, source, line, col, filename):
         out = []
         tmp_filecache = {}
         for c in completions:
-            out.append(self.parse_completion(c, tmp_filecache))
+            try:
+                out.append(self.parse_completion(c, tmp_filecache))
+            except Exception as ex:
+                log.error('exception: ', ex)
```
Comment (blueyed) [inline rplugin/python3/deoplete/sources/deoplete_jedi/server.py]: Please fixup:
```
In [4]: log = logging.getLogger(__name__)

In [5]: log.error("foo", "bar")
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.6/logging/__init__.py", line 992, in emit
    msg = self.format(record)
…
  File "<ipython-input-5-cbb29d08e14d>", line 1, in <module>
    log.error("foo", "bar")
Message: 'foo'
Arguments: ('bar',)
```

**Comment by blueyed** [comment]
@mallochine 
Can you provide a test case for this?

@Shougo 
You should not merge things that break it more in strange ways.
I assumed you want to see steps to reproduce it always?!

**Comment by Shougo** [comment]
I should revert the changes?

**Comment by Shougo** [comment]
> I assumed you want to see steps to reproduce it always?!

I have read the code and thought it is just simple fixes.
Sorry.

**Comment by Shougo** [comment]
Reverted.

**Comment by blueyed** [comment]
@mallochine 
Please shed some light.

**Comment by mallochine** [comment]
the problem was an exception occurred in self.parse_completion. The error was some encoding thing like utf8. Can we just have a %s there to fix it? I don't have time for a simple test case.

**Comment by Shougo** [comment]
> the problem was an exception occurred in self.parse_completion. The error was some encoding thing like utf8. Can we just have a %s there to fix it? I don't have time for a simple test case.

The test case is needed.

**Comment by mallochine** [comment]
OK how to add test case?

**Comment by Shougo** [comment]
Please provide the error cases.

* The minimal init.vim
* The minimal script
* The reproduce ways from neovim starting

**Pull request merged**